add-ons/debezium/README.md

```diff
@@ -0,0 +1,100 @@
+# Deploying Debezium on OpenShift
+
+This addon provides an easy way to install [Debezium](http://debezium.io/) on [Minishift](https://github.com/minishift/minishift).
+
+Prerequisites:
+
+ * [Installed and running](https://docs.openshift.org/latest/minishift/getting-started/installing.html) Minishift
+ * [Installed](https://docs.openshift.org/latest/minishift/command-ref/minishift_oc-env.html) OpenShift CLI
+
+## Start Minishift
+The deployment works with Minishift default resources - 2 CPUs and 2 GB RAM. Nevertheless we prefer to allocate more memory (4 GB) available to Debezium and associated infrastructure. Start Minishift using following command to provide more RAM.
+```
+$ minishift start --memory=4GB
+```
+
+## Add-on installation
+Clone this repository onto your local machine and then install the add-on via:
+```
+    $ minishift addons install <path_to_directory_containing_this_readme>
+```
+
+## Debezium start
+Deploy the Kafka broker, Kafka Connect with Debezium and MySQL connector
+
+```
+minishift addon apply -a DEBEZIUM_VERSION=0.6.1 -a DEBEZIUM_PLUGIN=mysql -a PROJECT=myproject debezium
+```
+
+After a while all parts should be up and running:
+
+```
+oc get pods
+NAME                    READY     STATUS      RESTARTS   AGE
+debezium-1-build        0/1       Completed   0          3m
+debezium-2-build        0/1       Completed   0          3m
+kafka-0                 1/1       Running     2          3m
+kafka-1                 1/1       Running     0          2m
+kafka-2                 1/1       Running     0          2m
+kafka-connect-3-3v4n9   1/1       Running     1          3m
+zookeeper-0             1/1       Running     0          3m
+```
+
+## Supported parameters
+* `DEBEZIUM_VERSION` - the verson of Debezium to be used (only released versions are supported)
+* `DEBEZIUM_PLUGIN` - which plugin should be added into the Connect cluster
+* `PROJECT` - the name of the project to which Debezium is deployed
+
+## Debezium Tutorial
+You can try the [Debezium Tutorial](http://debezium.io/docs/tutorial/) in this environment too. Yoy can either follow the steps below to dpeloy the test database or use a [prepared MiniShift add-on](https://github.com/debezium/debezium-examples/tree/master/openshift/tutorial-database) for that purpose.
+
+First we need to start a MySQL server instance:
+
+```
+# Deploy pre-populated MySQL instance
+oc new-app --name=mysql debezium/example-mysql:0.6
+
+# Configure credentials for the database
+oc env dc/mysql  MYSQL_ROOT_PASSWORD=debezium  MYSQL_USER=mysqluser MYSQL_PASSWORD=mysqlpw
+```
+
+A new pod with MySQL server should be up and running:
+
+```
+oc get pods
+NAME                             READY     STATUS      RESTARTS   AGE
+...
+mysql-1-4503l                    1/1       Running     0          2s
+mysql-1-deploy                   1/1       Running     0          4s
+...
+```
```

Comment:
At this state I am seeing one mysql pod, is it ok?

```
$ oc get pods
NAME                    READY     STATUS      RESTARTS   AGE
debezium-1-build        0/1       Completed   0          7m
debezium-2-build        0/1       Completed   0          6m
kafka-0                 1/1       Running     1          7m
kafka-1                 1/1       Running     0          6m
kafka-2                 1/1       Running     0          6m
kafka-connect-3-77dwj   1/1       Running     0          5m
mysql-2-kgz31           1/1       Running     0          1m
zookeeper-0             1/1       Running     0          7m
```

Reply:
Yes it is ok, I just took the pod list probably too early before the deployment was completed